pkgs/applications/networking/browsers/firefox/common.nix

```diff
@@ -88,8 +88,18 @@ stdenv.mkDerivation (rec {
     rm -f js/src/configure
     rm -f .mozconfig*
 
+ '' + lib.optionalString (stdenv.lib.versionAtLeast version "58.0.0") ''
+    cat >.mozconfig <<END_MOZCONFIG
+    ${lib.concatStringsSep "\n" (map (flag: "ac_add_options ${flag}") configureFlags)}
+    END_MOZCONFIG
+  '' + lib.optionalString googleAPISupport ''
+    # Google API key used by Chromium and Firefox.
+    # Note: These are for NixOS/nixpkgs use ONLY. For your own distribution,
+    # please get your own set of keys.
+    echo "AIzaSyDGi15Zwl11UNe6Y-5XW_upsfyw31qwZPI" >/tmp/ga
```

Comment:
sure you can just use /tmp here (looking at non-sandboxed builds)? I'd suggest using `mktemp -d` to be on the safe side

Reply:
Seems like you can use [$TMPDIR](https://nixos.org/nix/manual/#sec-common-env)